scene/resources/visual_shader_nodes.cpp

```diff
@@ -5213,12 +5213,22 @@ String VisualShaderNodeFloatParameter::get_output_port_name(int p_port) const {
 
 String VisualShaderNodeFloatParameter::generate_global(Shader::Mode p_mode, VisualShader::Type p_type, int p_id) const {
 	String code = "";
+	bool add_comma = true;
 	if (hint == HINT_RANGE) {
 		code += _get_qual_str() + "uniform float " + get_parameter_name() + " : hint_range(" + rtos(hint_range_min) + ", " + rtos(hint_range_max) + ")";
 	} else if (hint == HINT_RANGE_STEP) {
 		code += _get_qual_str() + "uniform float " + get_parameter_name() + " : hint_range(" + rtos(hint_range_min) + ", " + rtos(hint_range_max) + ", " + rtos(hint_range_step) + ")";
 	} else {
 		code += _get_qual_str() + "uniform float " + get_parameter_name();
+		add_comma = false;
+	}
+	if (get_qualifier() == QUAL_INSTANCE_INDEX) {
+		if (add_comma) {
+			code += ", ";
+		} else {
+			code += ": ";
+		}
+		code += vformat("instance_index(%d)", get_instance_index());
```

Comment:
```suggestion
		code += "instance_index(" + itos(get_instance_index()) + ")";
```
I don't think this needs `vformat`, for all